src/python/pants/bin/local_pants_runner.py

```diff
@@ -75,6 +79,7 @@ def _run(self):
         repro.capture(run_tracker.run_info.get_as_dict())
 
       # Record the preceding product graph size.
+      run_tracker.new_pre_timed_workunit('graph_warmth', self._graph_warmth_timing)
```

Comment:
Will this render under `main`? I think it would need to to actually be counted toward cumulative timings for `main`. Should be able to check easily in `./pants server --open`.

Reply:
apparently not: the self_timings show up implicitly as main:graph_warmth, but tests reveal that the cumulative_timings for main < self_timings for main:graph_warmth..